Framework/Algorithms/test/GetEiTest.h

```diff
@@ -289,11 +291,12 @@ class GetEiTestPerformance : public CxxTest::TestSuite {
   static void destroySuite(GetEiTestPerformance *suite) { delete suite; }
 
   void setUp() {
-    inputWS1 = createTestWorkspaceWithMonitors();
+	getEiTest = GetEiTest();
+    inputWS1 = getEiTest.createTestWorkspaceWithMonitors();
     outputName1 = "eitest1";
     AnalysisDataService::Instance().add(outputName1, inputWS1);
 
-    inputWS2 = createTestWorkspaceWithMonitors();
+    inputWS2 = getEiTest.createTestWorkspaceWithMonitors();
```

Comment:
Is it okay to do that, instead of re-writing the methods or separating them in a different class?